### DISCUSSION TITLE
How to change default asms track from  auto to other track names when player loads video in BP

### DISCUSSION THREAD

**Discussion by selormsedofia** [jhomlala/betterplayer#672] [Q&A]
Hello,

What is the best way to change default asms track name from auto to other tracks when player initializes?

Regards

**Comment by jhomlala** [comment]
Use translations which can be defined in `BetterPlayerConfiguration` to change preferred name of "auto". There's `qualityAuto` key in `BetterPlayerTranslations`.

**Comment by selormsedofia** [comment]
What i really want to know is is there a way to set the initial/default asms track that BP should play rather than the default auto track BP chooses when video loads?
